lib/scenic/view.rb

```diff
@@ -44,10 +44,10 @@ def ==(other)
     def to_schema
       materialized_option = materialized ? "materialized: true, " : ""
       <<-DEFINITION
-
   create_view :#{name}, #{materialized_option} sql_definition: <<-\SQL
     #{definition.indent(2)}
   SQL
+
       DEFINITION
```

Comment:
Does `strip_heredoc` let you write this without the reverse-indentation that breaks back to the left?